ProjectLighthouse.Localization/StringLists/ModerationStrings.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace LBPUnion.ProjectLighthouse.Localization.StringLists;
+
+public static class ModerationStrings
+{
+    public static readonly TranslatableString BannedHeading = create("banned_heading");
```

Comment:
The wording of some of these feels off. For these two you use banned as a verb then you use ban as a noun for the reason. I don't really mind which you use but I would keep them consistent.